controllers/common.go

```diff
@@ -550,3 +551,72 @@ func GetCommonRbacRules(privileged bool) []rbacv1.PolicyRule {
 
 	return []rbacv1.PolicyRule{rbacPolicyRule}
 }
+
+// Some frameworks like (e.g., Tobiko and Horizon) require password value to be
+// present in clouds.yaml. This code ensures that we set a default value of
+// 12345678 when password value is missing in the clouds.yaml
+func EnsureCloudsConfigMapExists(
+	ctx context.Context,
+	instance client.Object,
+	helper *helper.Helper,
+	labels map[string]string,
+) (ctrl.Result, error) {
+	const openstackConfigMapName = "openstack-config"
+	const testOperatorCloudsConfigMapName = "test-operator-clouds-config"
```

Comment:
(non-blocking) question: would take the name from pkg.util.common.go ? So we don't need to maintain same const in two different places.